src/pages/api/movies.ts

```
@@ -0,0 +1,29 @@
+  req: NextApiRequest,
+  res: NextApiResponse
+) {
+  const client = await clientPromise;
+  const db = client.db("movieDB"); // або ім'я твоєї бази
+
+  const collection = db.collection("movies"); // колекція для фільмів
+
+  switch (req.method) {
+    case "GET":
+      const movies = await collection.find({}).toArray();
+      res.status(200).json(movies);
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)